Fix error adding pod Webhook module comment

dafu-wu opened the pull request:
When I read the Add POD Webhook to HashMap module, I noticed a discrepancy between the comments and the code
```
// Currently, if **SidecarSet** is not installed, we can also disable pod webhook.
if !utildiscovery.DiscoverObject(&policyv1alpha1.**PodUnavailableBudget**{}) {
	return false
}
```
It confused me, so I changed it。

@furykerry

FillZpp commented:
@dafu-wu Thanks for fixing the comment. Some Nits:

1. DCO failed, which means you execute `git commit` without `-s` to sign yourself.
2. Looks like the commit has incorrect git config, may be your private git config.

So you might have to `git commit --amend -s --author="YOUR_GITHUB_NAME <YOUR_GITHUB_EMAIL>"` and then force push again.

FillZpp commented:
@dafu-wu Sorry, it seems you have not changed the git config yet.

Please first `git config user.name "YOUR_GITHUB_NAME"` and `git config user.email "YOUR_GITHUB_EMAIL"` in your local  kruise repo. Then `git commit --amend` and force push again.

dafu-wu commented:
> @dafu-wu Sorry, it seems you have not changed the git config yet.
> 
> Please first `git config user.name "YOUR_GITHUB_NAME"` and `git config user.email "YOUR_GITHUB_EMAIL"` in your local kruise repo. Then `git commit --amend` and force push again.

Thank you for your correction. I have corrected it。

FillZpp commented:
> Thank you for your correction. I have corrected it

Good, I have seen your correct git config.

However, you have submitted an additional `coverage.xml` file in your commit... Please remove it or add it into `.gitignore` and push again. Thanks.

FillZpp reviewed (approved):
/lgtm
Thanks for contributing!

dafu-wu commented:
> > Thank you for your correction. I have corrected it
> 
> Good, I have seen your correct git config.
> 
> However, you have submitted an additional `coverage.xml` file in your commit... Please remove it or add it into `.gitignore` and push again. Thanks.

I‘m sorry for that, I have corrected it, I might not be myself today 😊
